env.yml

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
This file shouldn't exist

Reply:
Why not?

Reply:
Because the datasets directory does not exist anymore